contracts/dtoken/src/interest_model.rs

```diff
@@ -19,6 +26,7 @@ impl Contract {
     }
 
     fn get_util(&self, underlying_balance: WBalance, total_borrows: WBalance, total_reserves:WBalance) -> Ratio{
+        assert!((Balance::from(underlying_balance) + Balance::from(total_borrows) - Balance::from(total_reserves)) > 0, "Cannot calculate utilization rate as contract has no assets");
```

Comment:
If there are no assets, it means that util rate = 0. So we need to discuss this flow - not sure that assert good approach in this case.